### DISCUSSION TITLE
Feature request: Audio quality switcher

### DISCUSSION THREAD

**Discussion by simplybloxxed** [rustdesk/rustdesk#1198] [Ideas]
Alright, I installed rustdesk quite literally today, and it works solidly for a decent amount of my general purposes (I'm actually using it right now to write this!) however it does have one major problem: The audio latency, the 2-3 second delay may be fine for tech support or quickly checking a system, but in my case the delay makes things like gaming and editing is absolutely awful. It'd be nice if you could switch inbetween both, as it'll help the use case of rustdesk even further. Need higher quality audio for whatever reason? Just switch it over, and switch it back when youre loading up a game. Perfect for all use cases.

**Comment by rustdesk** [comment]
@Heap-Hop take note

**Comment by patryk-siewiera** [comment]
bump, I need this!
